pkg/compute/init.go

```diff
@@ -151,13 +151,14 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 	c.path = abspath
 
 	if name == "" {
-		name = filepath.Base(c.path)
-		fmt.Fprintf(progress, "--name not specified, using %s\n\n", name)
```

Comment:
Would be good to keep the verbose output here.

Reply:
None of the other options that have defaults show this and the message is incorrect since this is not necessarily the name you will end up using so I think removing this is better.